joomla/joomla-cms
# backport #35574 (Fix wrong error message for user activation)

This PR is to Backport #35574 - I hope it's done correctly

Pull Request for Issue #35849 .

### Summary of Changes
Copied the code from Joomla 4 and added Js ;-)


### Testing Instructions
Testing Instructions

1.     Set registration to user activation
2.     Register a new user
3.     click on the verification link
4.     click again on the verification link

Actual result BEFORE applying this Pull Request

![grafik](https://user-images.githubusercontent.com/828371/141767862-f7d35f29-dfb1-4676-924e-eb8c0bd5bc29.png)
Expected result AFTER applying this Pull Request

User is redirected to the login (with the same message as normal message)
